### DISCUSSION TITLE
Merging PRs from upstream

### DISCUSSION THREAD

**Discussion by robamu** [spacefisch/stm32-cmake#2] [General]
I guess this is t he communication feature of GitHub so I am just going to use this. I think this fork could be used to merge all the good PRs from upstream which have not been merged yet, as long as the upstream continues to appear unmaintained. @g-arjones  can you push your changes here as well? Or should I just add your fork as a remote and create the branches?

**Comment by g-arjones** [comment]
> Or should I just add your fork as a remote and create the branches?

Sounds good.

I really appreciate the initiative. Thanks.
